web/src/user_group_pill.ts

```diff
@@ -17,8 +18,12 @@ export type UserGroupPillData = UserGroup & {
 };
 
 function display_pill(group: UserGroup): string {
-    const group_members = user_groups.get_recursive_group_members(group);
-    return `${user_groups.get_display_group_name(group)}: ${group_members.size} users`;
+    const group_members = get_group_members(group);
+
+    // In case of 0, which is technically possible in case all the
+    // users in a group are deactivated, we will show 0 users.
+    const user_pluralized_text = group_members.length === 1 ? "user" : "users";
+    return `${user_groups.get_display_group_name(group)}: ${group_members.length} ${user_pluralized_text}`;
```

Comment:
This was a preexisting issue, but this is not valid internationalization. We need to use the ICU plurals syntax for this.

See `git grep ' plural, ' web/src/` and the discussion on plurals in https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details.

Reply:
Fixed in 213ce76f0598ef67ccf03f5b1cba21e690b3d313